### PR TITLE
[script.module.pysocks] 1.6.8

### PR DESCRIPTION
**Pull request by dagwieers**
### Description
Various modules use Requests for doing HTTP communication, but this does not support the global Kodi proxy settings when using SOCKS. I have been adding proxy support to various addons (e.g. [inputstreamhelper](https://github.com/emilsvennesson/script.module.inputstreamhelper/pull/51), [vrt.nu](https://github.com/pietje666/plugin.video.vrt.nu/pull/87/files)), and am looking to add it to more addons, however to support SOCKS with Requests we need this Python library available.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.pysocks/addon.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pysocks" name="PySocks" version="1.6.8" provider-name="anorov">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```
Comment (Rechi) [inline script.module.pysocks/addon.xml]: > WARN: For krypton it is advised to set xbmc.python version to 2.25.0

Comment (dagwieers) [reply script.module.pysocks/addon.xml]: Fixed.

**Comment by dagwieers** [comment]
@Rechi The next release of **plugin.video.vrt.nu** will require this. And we have a PR open for **script.module.inputstreamhelper**. I hope to also improve **plugin.video.netflix**.

PS I count 52 modules in repo-plugins that us Requests without support proxies (actually no addon currently supports the Kodi proxy settings) and I hope we can change that by showing these addons how easy it is.

**Comment by dagwieers** [comment]
@Rechi We released VRT.NU v1.7.0 which makes use of this. Can you please look at https://github.com/xbmc/repo-plugins/pull/2349 and merge it as soon as possible. Or users have been affected today because the currently released addon (v1.6.0) stopped working.

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Please squash the commits.

**Comment by dagwieers** [comment]
Done.

(Just to understand, is there any reason why "squash and merge" in GitHub is not an option ?)

**Pull request merged**